app/views/audit.scala.html

```diff
@@ -174,7 +174,7 @@ <h4 class="survey-question">@Html(Messages("audit.survey." + surveyQuestion.surv
                         <img src='@routes.Assets.at("javascripts/SVLabel/img/icons/noun_lost_1779388.png")'  alt="Stuck icon">
                         @Html(Messages("audit.left.ui.stuck"))
                     </div>
-                    <div id="left-column-feedback-button" class="button">
+                    <div id="left-column-feedback-button" class="button" data-container="body" data-toggle="popover" data-template="<div class='feedback-popover' role='tooltip'><div class='arrow'></div><div class='popover-content'></div></div>" data-placement="top" data-content='@Messages("feedback.submitted")' data-trigger = "manual">
```

Comment:
super nit-picky, but can you switch the `data-trigger = "manual"` to `data-trigger="manual"` to fit the HTML convention?